search/routes/api/__init__.py

```diff
@@ -39,12 +39,18 @@ def search() -> Response:
     # if requested == ATOM_XML:
     #     return serialize.as_atom(data), status, headers
     response_data = serialize.as_json(data['results'], query=data['query'])
-    return response_data, status_code, headers # type: ignore
+    
+    headers.update({'Content-type': f'{JSON}; charset=utf-8'})
```

Comment:
Any particular reason not to include the charset part in ``JSON``?

Reply:
Fixed